2_task/arrayOperation.js

```diff
@@ -0,0 +1,158 @@
+//function isArray
+function isArrayByInstance(arr) {
+  return arr instanceof Array;
+}
+
+function isArrayByPrototype(arr) {
+  return Object.prototype.toString.call(arr) === '[object Array]';
+}
+
+function isArrayByDuckTyping(arr) {
+  if (arr.splice) {
+    return true;
+  }
+  return false;
+}
+//function range
+function createArray(start, end, multiplay) {
```

Comment:
could you move code from this func to range? btw it looks a bit complicated for this.
PS you have right direction